### PR TITLE
(dev/core#491) campaign cleanup for 'pledged but not paid' report and…

### PR DESCRIPTION
**Pull request by yashodha**
… show disabled campaigns as well.

Overview
----------------------------------------
Centralize and cleanup code for campaigns in reports and show disabled campaigns as well

Before
----------------------------------------

pbnp_before

After
----------------------------------------
pbnp_after

Comments
----------------------------------------
Inactive campaigns can be seen in report filter and results as well.

### CODE REVIEW

**Comment by yashodha** [comment]
test this please

File: CRM/Report/Form/Pledge/Pbnp.php [hunk 1]
```diff
@@ -49,12 +49,6 @@ public function __construct() {
 
     // Check if CiviCampaign is a) enabled and b) has active campaigns
     $config = CRM_Core_Config::singleton();
```
Comment (seamuslee001) [inline CRM/Report/Form/Pledge/Pbnp.php]: @yashodha L50 and L51 can probably be deleted too i think and maybe L52 as well

**Comment by seamuslee001** [comment]
Tested the report it works as expected and outputs correct information. Just a minor cleanup apart from that this is good to be merged

**Comment by seamuslee001** [comment]
Tested the report and it works fine merging

**Pull request merged**